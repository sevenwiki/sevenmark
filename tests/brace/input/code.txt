{{{#code #lang="rust"
fn main() {
    println!("Hello, world!");
}
}}}