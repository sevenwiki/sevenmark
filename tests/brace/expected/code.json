[
  {
    "CodeElement": {
      "location": {
        "start": 0,
        "end": 70
      },
      "language": [
        {
          "Text": {
            "location": {
              "start": 16,
              "end": 20
            },
            "content": "rust"
          }
        }
      ],
      "content": [
        {
          "Text": {
            "location": {
              "start": 22,
              "end": 32
            },
            "content": "fn main() "
          }
        },
        {
          "Text": {
            "location": {
              "start": 32,
              "end": 33
            },
            "content": "{"
          }
        },
        {
          "Text": {
            "location": {
              "start": 33,
              "end": 65
            },
            "content": "\n    println!(\"Hello, world!\");\n"
          }
        },
        {
          "Text": {
            "location": {
              "start": 65,
              "end": 66
            },
            "content": "}"
          }
        },
        {
          "Text": {
            "location": {
              "start": 66,
              "end": 67
            },
            "content": "\n"
          }
        }
      ]
    }
  }
]